app/Makefile

```diff
@@ -40,7 +40,7 @@ DEFINES += PRODUCTION_BUILD=$(PRODUCTION_BUILD)
 
 include $(CURDIR)/../deps/ledger-zxlib/makefiles/Makefile.app_testing
 
-DEFINES += REVIEW_SCREEN_ENABLED SHORTCUT_MODE_ENABLED
```

Comment:
Not sure if we should remove the REVIEW_SCREEN_ENABLE this will lead to the removal of the "Review Address" screen on the show address command.